docker-compose.yaml

```
@@ -0,0 +1,76 @@
+services:
+  mariadb:
+    image: mariadb:11.5.2
+    environment:
+      MARIADB_ALLOW_EMPTY_ROOT_PASSWORD: "0"
+      MARIADB_DATABASE: naad_connector
+      MARIADB_ROOT_PASSWORD: ${MARIADB_ROOT_PASSWORD}
+    ports:
+      - "3306:3306"
+    volumes:
+      - database-volume:/var/lib/mysql
+    networks:
+      - naad_deploy
+    healthcheck:
+      test: ["CMD-SHELL", "mariadb-admin -u root -p${MARIADB_ROOT_PASSWORD} ping -h 127.0.0.1"]
+      interval: 10s
+      timeout: 5s
+      retries: 3
+    restart: always
+
+  # wait for mariadb to start up
+  database-migrate:
+    image: bcgovgdx/naad-app
+    environment:
+      MARIADB_ROOT_PASSWORD: ${MARIADB_ROOT_PASSWORD} # Loaded from .env
+      MARIADB_SERVICE_HOST: mariadb
+      MARIADB_SERVICE_PORT: 3306
+    depends_on:
+      mariadb:
+        condition: service_healthy
+    entrypoint: ["vendor/bin/doctrine-migrations", "-vvv", "-n", "migrate"]
+    networks:
+      - naad_deploy
+
+  # wait for database migration to complete
+  naad-app:
+    image: bcgovgdx/naad-app
+    environment:
+      DESTINATION_URL: https://0.0.0.0/wp-json/naad/v1/alert
```

this should have an env variable 
```suggestion
      DESTINATION_URL: ${DESTINATION_URL:-https://0.0.0.0/wp-json/naad/v1/alert}
```